### ISSUE TITLE
Add option to disable automatic journal operations from e.g. Make a Move

### ISSUE THREAD

**Issue by zkat** [cwegrzyn/iron-vault#123]
to go with our larger story of folks being able to pick and choose the parts of Iron Vault they want to use, we should make the whole "we'll append stuff to your journal" bit optional, configurable through a setting.

I don't think this needs to be granular. It can be either all-in or not-at-all.

**Comment by cwegrzyn** [comment]
Hmm, this makes sense for some of the "create an x" commands. But right now the only value of "make a move" _is_ the journal entry. Or are you thinking we pop up a dialog with results?

**Comment by zkat** [comment]
Yep, simply show a dialog with results (or the next "screen" of the modal or whatever)
